package.json

```diff
@@ -38,10 +39,28 @@
     "eslint": "^6.7.2",
     "eslint-plugin-prettier": "^3.3.1",
     "eslint-plugin-vue": "^7.0.0",
+    "husky": "^7.0.2",
+    "lint-staged": "^11.1.2",
     "node-sass": "^4.12.0",
     "prettier": "^2.2.1",
     "sass-loader": "^8.0.2",
     "typescript": "3.9.10",
     "vue-jest": "^5.0.0-0"
-  }
+  },
+  "lint-staged": {
+    "*.{html,vue,css,scss}": [
+      "prettier --write",
+      "git add"
+    ],
+    "*.{js,ts,vue}": [
+      "eslint --fix"
+    ],
+    "*": "prettier --ignore-unknown --write",
+    "*.{js,ts}": "eslint --cache --fix"
+  },
+  "husky": {
+    "hooks": {
+    "pre-commit": "lint-staged && yarn run test:unit"
```

Comment:
remove '&& yarn run test:unit'
don't need to run test as well we don't use yarn